app/app.go

```diff
@@ -0,0 +1,5 @@
+package app
+
+const (
+	AppVersion = "RC-1.0.0.alpha" // DISCUSS (team): see app/pocket/main.go, one of the two should be a source of truth, potentially replaced at build-time as described in `docs/build/README.md`
```

Comment:
@iajrz can you confirm the versioning schema?

Reply:
I updated here a little while ago: https://github.com/pokt-network/pocket/blob/main/docs/build/README.md#release-tag-versioning.

The currently release is `v0.0.1-pre-alpha.1 ` so we should use that.

Reply:
@iajrz bump on this